extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/PanacheRepositoryBase.java

```diff
@@ -634,6 +634,17 @@ public default long deleteAll() {
         throw JpaOperations.implementationInjectionMissing();
     }
 
+    /**
+     * Delete an entity of this type by ID.
+     *
+     * @param id the ID of the entity to delete.
+     * @return false if the entity was not deleted (not found).
+     */
+    @GenerateBridge
+    public default boolean deleteById(Object id) {
```

Comment:
Shouldn't this be of type `Id` for coherence with `findById(Id)` ?

Reply:
done